tools/wavetool.py

```diff
@@ -0,0 +1,279 @@
+#! /usr/bin/python3
+
+import os
+import sys
+import argparse
+import numpy as np
+import scipy.signal as signal
+import scipy.io.wavfile as wavefile
+import scipy.fftpack as fftpack
+
+eps = 0.0000000001 # used to avoid zero division error
+
+class WavGenerator:
```

Comment:
I don't see the point of a class with only @staticmethods. No need to indent these functions in a class, just move them to the top-level.